docs_user/modules/proc_deploying-backend-services.adoc

```diff
@@ -92,43 +84,40 @@ make input
 ----
 endif::[]
 
-. If the `$ADMIN_PASSWORD` is different than the already set password
-in `osp-secret`, amend the `AdminPassword` key in the `osp-secret`
-correspondingly:
+. If the `$ADMIN_PASSWORD` is different than the password you set
+in `osp-secret`, amend the `AdminPassword` key in the `osp-secret`:
 +
 ----
-oc set data secret/osp-secret "AdminPassword=$ADMIN_PASSWORD"
+$ oc set data secret/osp-secret "AdminPassword=$ADMIN_PASSWORD"
 ----
 
 . Set service account passwords in `osp-secret` to match the service
 account passwords from the original deployment:
 +
 ----
-oc set data secret/osp-secret "AodhPassword=$AODH_PASSWORD"
-oc set data secret/osp-secret "BarbicanPassword=$BARBICAN_PASSWORD"
-oc set data secret/osp-secret "CeilometerMeteringSecret=$CEILOMETER_METERING_SECRET"
-oc set data secret/osp-secret "CeilometerPassword=$CEILOMETER_PASSWORD"
-oc set data secret/osp-secret "CinderPassword=$CINDER_PASSWORD"
-oc set data secret/osp-secret "GlancePassword=$GLANCE_PASSWORD"
-oc set data secret/osp-secret "HeatAuthEncryptionKey=$HEAT_AUTH_ENCRYPTION_KEY"
-oc set data secret/osp-secret "HeatPassword=$HEAT_PASSWORD"
-oc set data secret/osp-secret "IronicPassword=$IRONIC_PASSWORD"
-oc set data secret/osp-secret "IronicInspectorPassword=$IRONIC_PASSWORD"
-oc set data secret/osp-secret "ManilaPassword=$MANILA_PASSWORD"
-oc set data secret/osp-secret "NeutronPassword=$NEUTRON_PASSWORD"
-oc set data secret/osp-secret "NovaPassword=$NOVA_PASSWORD"
-oc set data secret/osp-secret "OctaviaPassword=$OCTAVIA_PASSWORD"
-oc set data secret/osp-secret "PlacementPassword=$PLACEMENT_PASSWORD"
-oc set data secret/osp-secret "SwiftPassword=$SWIFT_PASSWORD"
+$ oc set data secret/osp-secret "AodhPassword=$AODH_PASSWORD"
+$ oc set data secret/osp-secret "BarbicanPassword=$BARBICAN_PASSWORD"
+$ oc set data secret/osp-secret "CeilometerMeteringSecret=$CEILOMETER_METERING_SECRET"
+$ oc set data secret/osp-secret "CeilometerPassword=$CEILOMETER_PASSWORD"
+$ oc set data secret/osp-secret "CinderPassword=$CINDER_PASSWORD"
+$ oc set data secret/osp-secret "GlancePassword=$GLANCE_PASSWORD"
+$ oc set data secret/osp-secret "HeatAuthEncryptionKey=$HEAT_AUTH_ENCRYPTION_KEY"
+$ oc set data secret/osp-secret "HeatPassword=$HEAT_PASSWORD"
+$ oc set data secret/osp-secret "IronicPassword=$IRONIC_PASSWORD"
+$ oc set data secret/osp-secret "IronicInspectorPassword=$IRONIC_PASSWORD"
+$ oc set data secret/osp-secret "ManilaPassword=$MANILA_PASSWORD"
+$ oc set data secret/osp-secret "NeutronPassword=$NEUTRON_PASSWORD"
+$ oc set data secret/osp-secret "NovaPassword=$NOVA_PASSWORD"
+$ oc set data secret/osp-secret "OctaviaPassword=$OCTAVIA_PASSWORD"
+$ oc set data secret/osp-secret "PlacementPassword=$PLACEMENT_PASSWORD"
+$ oc set data secret/osp-secret "SwiftPassword=$SWIFT_PASSWORD"
 ----
 
-. Deploy `OpenStackControlPlane`. Make sure to only enable DNS,
+. Deploy the `OpenStackControlPlane` CR. Ensure that you only enable the DNS,
```

Comment:
Can we provide the command for this? I'm not sure if something like the following is correct:
$ oc patch openstackcontrolplane openstack --type=merge --patch '
spec:
  dns:
      enabled: true
      apiOverride:
          route: {}
…

$ oc patch openstackcontrolplane openstack --type=merge --patch '
spec:
 MariaDB:
      enabled: true
      apiOverride:
          route: {}
…
and so on...

Reply:
The command for the creation of the OpenStackControlPlane CR is below, it's the one that starts:

```
oc apply -f - <<EOF
apiVersion: core.openstack.org/v1beta1
kind: OpenStackControlPlane
...
```

Over time these TLS-specific configuration amendments were added prior to it, so this sentence got shifted out of place. It should immediately precede the `oc apply` command.